packages/amplify-category-notifications/src/commands/notifications/configure.ts

```diff
@@ -26,14 +26,14 @@ export const alias = 'update';
  */
 export const run = async (context: $TSContext): Promise<$TSContext> => {
   const availableChannelViewNames = getAvailableChannelViewNames();
-  const channelName = context.parameters.first;
-  let channelViewName = channelName ? getChannelViewName(channelName) : undefined;
+  let channelViewName = context.parameters.first ? getChannelViewName(context.parameters.first) : undefined;
 
   if (!channelViewName || !availableChannelViewNames.includes(channelViewName)) {
     channelViewName = await prompter.pick('Choose the notification channel to configure', availableChannelViewNames);
   }
-  if (channelViewName && typeof channelName === 'string') {
-    const selectedChannel = getChannelNameFromView(channelViewName);
+
+  if (channelViewName) {
+    const channelName = getChannelNameFromView(channelViewName);
     let pinpointAppStatus = await pinpointHelper.ensurePinpointApp(context, undefined);
```

Comment:
what's the type of `channelName` if not `string` and should that always be the same type for all channels?

Reply:
the problem is that the channelName is not passed as argument, it needs to be obtained from the channelViewName prompted